### PR TITLE
[PC-1274] Full stops in descriptions plus grammar improvements and typos fixes

### PR DESCRIPTION
**Pull request by jcarolinares**
## What This PR Changes
- Full stops in descriptions plus grammar improvements and typos fixes

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/software/plc-ide/tutorials/can-setup/content.md [hunk 1]
```diff
@@ -137,5 +137,5 @@ You can edit the table by adding, removing new variables, you can also assign an
 ### Next Steps
 
 Now that you have completed the setup of the Portenta Machine Control and its CAN bus communication with the Arduino PLC IDE, you can continue with:
-- Plug a CAN sensor to your device and get its data.
+- Plug a CAN sensor into your device and get its data.
```
Comment (jhansson-ard) [inline content/software/plc-ide/tutorials/can-setup/content.md:140]: I think saying "to" here is correct, but I could be wrong

File: content/software/plc-ide/tutorials/plc-programming-introduction/content.md [hunk 2]
```diff
@@ -82,19 +82,19 @@ There are 4 Tasks:
...
+If you want to configure the task, right-click on it and press `Task configuration`, you will be able to, for example, change the cycle time of the task.
 
 ***The execution order will be from top to bottom, the order shown inside each task.***
 
 ### Watch Variables
 
 To check that our program is running and counting correctly, we will use the **watch** feature which allows you to see the value of a variable in real time.
 
-To do so you need to go **View > Tool windows > Watch** and you will have a new tool window.
+To do so you need to go to **View**** > Tool** windows > Watch** and you will have a new tool window.
```
Comment (jhansson-ard) [inline content/software/plc-ide/tutorials/plc-programming-introduction/content.md:97]: What happened with the asteriskes here?

File: content/software/plc-ide/tutorials/plc-programming-introduction/content.md [hunk 3]
```diff
@@ -160,14 +160,14 @@ Here is a video doing that from scratch:
 <iframe width="100%" height="415" src="https://www.youtube.com/embed/0EdATSgIc9o" title="YouTube video player" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
 ### Sequential Function Chart
-This languages approaches the script similar to a flow-chart so you have blocks which do some work, you check the result of that work, and afterwards you keep flowing with the choices.
+This language approaches the script in a similar way as a flow-chart so you have blocks that do some work, you check the result of that work, and afterwards you keep flowing with the choices.
```
Comment (jhansson-ard) [inline content/software/plc-ide/tutorials/plc-programming-introduction/content.md:163]:
```suggestion
This language approaches the script in a similar way as a flowchart, meaning you have blocks that do some work, you check the result of that work, and afterward you keep flowing with the choices.
```

**Review by jhansson-ard — COMMENTED** [review]
Looks good, but please check my comments 🙏🏼

File: content/software/plc-ide/tutorials/can-setup/content.md (hunk 1 above)
Comment (jcarolinares) [reply content/software/plc-ide/tutorials/can-setup/content.md:140]: Recovered

File: content/software/plc-ide/tutorials/plc-programming-introduction/content.md (hunk 2 above)
Comment (jcarolinares) [reply content/software/plc-ide/tutorials/plc-programming-introduction/content.md:97]: Fixed, be careful with Grammarly sometimes it mess with *****

**Pull request merged**